app/channels/application_cable/channel.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
:clap: I think rubocop ask you for that.
Here (in Wolox), we are still not using that but in the futur we will implement it and needed for ruby 3. Just for the info ;)